eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -270,6 +270,12 @@
           "code" : "java.method.removed",
           "old" : "method void com.mysql.cj.protocol.ServerSession::setOldStatusFlags(int)",
           "justification" : "Fix method removed."
+        },
+        {
+          "regex": true,
+          "code" : "java.class.externalClassExposedInAPI",
+          "new" : "(class|enum) com.azure.ai.openai.(OpenAIAsyncClient|OpenAIClient|OpenAIClientBuilder|OpenAIServiceVersion|models.*)",
+          "justification" : "external class in API"
         }
```

Comment:
Still need this? There should be one rule in this file to allow SDK client exposed in API, could you find that rule and add this to it? We should avoid adding new rules.